Mathlib/Algebra/Quaternion.lean

```diff
@@ -744,8 +744,6 @@ namespace Quaternion
 
 variable {S T R : Type*} [CommRing R] (r x y : R) (a b : ℍ[R])
 
-export QuaternionAlgebra (re imI imJ imK)
```

Comment:
I'm guessing this is a bad merge. Could someone try reverting this file? (and, if that's okay, pushing also to nightly-testing)

Reply:
Reverting causes errors. So I think this diff is correct.